README.md

```diff
@@ -1,7 +1,4 @@
-# StackStorm Enterprise RBAC Backend for StackStorm Enterprise Edition
-
-StackStorm RBAC backend for enterprise version which contains all the proprietary RBAC
-(permission resolving) business logic.
+# RBAC Backend for StackStorm
```

Comment:
Does the note below this need an update? As it seems to relate to the enterprise code referring to open source code?

Reply:
I added a sentence to briefly mention history of this repo for clarification.